packages/liveblocks-redux/src/index.ts

```diff
@@ -289,13 +289,13 @@ const internalEnhancer = <T>(options: {
         client.leave(roomId);
       }
 
-      function newDispatch(action: any, state: any) {
+      function newDispatch(action: any) {
         if (action.type === ACTION_TYPES.ENTER) {
           enterRoom(action.roomId);
         } else if (action.type === ACTION_TYPES.LEAVE) {
           leaveRoom(action.roomId);
         } else {
-          store.dispatch(action, state);
```

Comment:
This seems to be a remnant of a past iteration? Going from the Redux type definitions, it looks like dispatch() only takes a single argument, not two. This may just have been passing implicit-undefined around in previous versions.